### PR TITLE
fix: p-chips do not work with reactive forms

### PR DESCRIPTION
**Pull request by vmuresanu**
### Defect Fixes
Fixes https://github.com/primefaces/primeng/issues/14062

### Changes
Added documentation for max items functionality

### CODE REVIEW

File: src/app/showcase/doc/chips/maxvaluesdoc.ts [hunk 1]
```diff
@@ -0,0 +1,46 @@
+import { Component, Input } from '@angular/core';
+import { FormControl } from '@angular/forms';
+import { Code } from '../../domain/code';
+
+@Component({
+    selector: 'max-values-doc',
+    template: ` <section class="py-4">
+        <app-docsectiontext [title]="title" [id]="id">
+            <p>Chips can have a maximum number of entered items. In order to set the limit <i>max</i> property is used and accepts a numeric value which reflects the maximum number of items in the chip list</p>
```
Comment (SoyDiego) [inline src/app/showcase/doc/chips/maxvaluesdoc.ts:9]: I would like to do a suggestion here:
`Chips can have a maximum number of entered items. To set this limit, the <i>max</i> property is used, which accepts a numeric value that represents the maximum number of items in the chip list`

Comment (vmuresanu) [reply src/app/showcase/doc/chips/maxvaluesdoc.ts:9]: Sounds good, will do the change

**Comment by cetincakiroglu** [comment]
Hi,

Thanks a lot for the contribution, we appreciate the support!

**Comment by vmuresanu** [comment]
Hello, anytime :)

**Pull request merged**